mac/ide-tour.md

```diff
@@ -110,7 +110,7 @@ For more information on using version control in Visual Studio, see the [Version
 ## Next steps
 
 - [Install Visual Studio for Mac](installation.md)
-- [Review the available workloads](/visualstudio/mac/workloads/)
+- [Review the available workloads](/mac/workloads.md)
```

Comment:
Since it's in the same folder, it could also be a relative link (that will also work on GitHub)
```suggestion
- [Review the available workloads](workloads.md)
```

Reply:
This is the correct form